packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/index.js

```diff
@@ -17,6 +18,7 @@ export default {
   ...agentParts,
   ...curatedListParts,
   ...localityParts,
+  ...storageParts,
```

Comment:
did you need this for something specific? if not we can remove it, because we don't need to import more than necessary (I think the accordion for physical objects is importing them, and we don't have any independent formParts from storage in the specimen form, so it shouldn't be needed here.) I understand if it looked to you that it was missing!

Reply:
Yes, it will not working if not exported from here. It said something like missing export or import, break from createAsyncView or somewhere wrapInColumn

Reply:
@antonoberg you can revert this 08e9536681f1e707af910c01bc1a58fe3e20898f and fix the circular dependency 🙏